app/assets/config/manifest.js

```diff
@@ -0,0 +1,2 @@
+//= link_tree ../images
+//= link_directory ../stylesheets .css
```

Comment:
1. What is this file about?
2. What does `link_tree` means?
3. What does `link_directory` means?

Reply:
1.  Any files or directories that are linked here in this manifest will be added to the pipeline for precompilation.
2. `link_tree` refers to all files within a folder and subfolders.
3. `link_directory` refers to only the files within the named directory.

Reply:
> Is this sort of a middle point for importing all the different types of assets?

Before answering your this question, can you tell me what you know about sprockets?

Reply:
Sprockets compiles and serve web assets. It allows you to organize an application’s JS & CSS files into smaller more manageable chunks that can be distributed over a number of directories and files. Sprockets then loads files specified, processes them if necessary, concatenates them into one single file, and then compresses them for better performance.

Reply:
sprockets use this file to determine how to compile.
So, does this answer your this question:
> Is this sort of a middle point for importing all the different types of assets?